src/2_banner.js

```diff
@@ -154,8 +154,11 @@ banner = function(branch, options, linkData, storage) {
 			}, banner_utils.animationSpeed + banner_utils.animationDelay);
 
 			setTimeout(function() {
-				if (options.position == 'top') { document.body.style.marginTop = '0px'; }
-				else if (options.position == 'bottom') { document.body.style.marginBottom = '0px'; }
+				if (options.position == 'top') {
+					document.body.style.paddingTop = '0px';
+					document.body.style['backgroundPositionY'] = '0px';
```

Comment:
hmm, i think this could break something, if you have your backgroundPositionY set to somethign like 'center'... – why do we need to change it?

also - maybe we should set both marginTop and paddingTop to 0?

also - maybe we shoudl save the previous values so we can restore them when the banner is hidden? I don't want to interfere with the CSS too much though so we should have a toggle switch to turn that off if we do it... sooo many 
switches :-/